src/types/chain.rs

```diff
@@ -99,3 +104,90 @@ impl From<BdkSyncResponse> for Update {
         Update(result.into())
     }
 }
+
+/// Represents the observed position of some chain data.
+#[wasm_bindgen]
+pub struct ChainPosition(BdkChainPosition<BdkConfirmationBlockTime>);
+
+impl Deref for ChainPosition {
+    type Target = BdkChainPosition<BdkConfirmationBlockTime>;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+#[wasm_bindgen]
+impl ChainPosition {
+    /// Returns whether [`ChainPosition`] is confirmed or not.
+    #[wasm_bindgen(getter)]
+    pub fn is_confirmed(&self) -> bool {
+        self.0.is_confirmed()
+    }
+
+    #[wasm_bindgen(getter)]
+    pub fn variant(&self) -> String {
+        if self.0.is_confirmed() {
+            "Confirmed".into()
+        } else {
+            "Unconfirmed".into()
+        }
```

Comment:
Would be great to use a string-enum here too, so the ts-type would be `'confirmed' | 'unconfirmed'`, WDYT?

Something like we do for `Network` for example:

```rust
/// The [`ChainPosition`] variant/status.
#[wasm_bindgen]
#[derive(Debug, Clone, Copy)]
pub enum ChainPositionVariant {
    /// Confirmed.
    Confirmed = "confirmed",
    /// Unconfirmed.
    Unconfirmed = "unconfirmed",
}
```

Reply:
Actually I implemented this `variant` because I thought it would be useful but I realize that not only it is not part of  BDK's API, but it's pretty useless since we can use `is_confirmed`.

I added it trying to somehow map to Rust's enums but that's just impossible in TypeScript. Removing.